packages/cli/src/templates/settings.template.toml

```diff
@@ -1,5 +1,8 @@
-[security]
-secret = 'YOUR_SECRET_HERE'
+# Permanent seeder path (will be completed on config:init)
```

Comment:
I think the hook variable should be: `endpoint-hook.js ` instead of `permanent-seeder-...`. At least in constants.js we are calling it `endpoint-hook.js`

Reply:
Yes, my bad.